lib/parsers/tiff_parser.rb

```diff
@@ -13,8 +13,8 @@ def call(io)
     return if !endianness || cr2_check(io)
 
     w, h = read_tiff_by_endianness(io, endianness)
-    scanner = FormatParser::EXIFParser.new(:tiff, io)
-    scanner.scan_image_exif
+    scanner = FormatParser::EXIFParser.new(io)
+    scanner.scan_image_tiff
```

Comment:
could we add a line of comment here that this method does not catch the `EXIFR::MalformedTIFF` exception on purpose?

Reply:
Yep

Reply:
Actually on second thought - this was not in the code previously, and we don't have any example files with invalid EXIF in a TIFF, so I'll leave it as is for now.

Reply:
👍 either way is good for me. just calling a method for throwing an exception might be a bit cryptic without comments for future maintainers.